src/openlmis-table/openlmis-table-container-shortener.scss

```diff
@@ -0,0 +1,17 @@
+.ps__scrollbar-x{
```

Comment:
I'm unsure how I feel about this, as this will conflict with the styles in the batch requisition screen with virtual scrolling.

Please prefix these CSS styles with openlmis-table-container (or something)